Data/Sys/GameSettings/GQPE78.ini

```diff
@@ -0,0 +1,12 @@
+# GQPE78 - SpongeBob SquarePants: Battle for Bikini Bottom
+
+[OnFrame_Enabled]
+# This game renders an EFB copy with texture repeating enabled
+# and it draws from texture coordinate 0.00195313 to 1.00195.
+# This only works on console and 1x IR due to low precision.
+# CopyRegionFix adjusts the region to not cause bugs at higher
```

Comment:
Shouldn't this say `EFBCopyFix`?

Also, EFBCopyFix is a slightly strange name in that it doesn't have any spaces. Normally we do use spaces in patch names.